cabal.project

```diff
@@ -12,8 +12,6 @@ index-state: 2024-06-10T12:08:58Z
 tests: True
 test-show-details: direct
 
--- benchmarks: True
```

Comment:
Removing because we have `benchmarks: True` / `benchmarks: False` in the if/else section below.